packages/esm-ward-app/src/hooks/useInpatientRequest.ts

```diff
@@ -0,0 +1,14 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import type { InpatientRequest } from '../types';
+
+//The api is still in implementation stage and this is just a placeholder for the api
+export function useInpatientRequest(locationUuid: string) {
+  const apiUrl = `${restBaseUrl}/emrapi/disposition?type=ADMISSION&dispositionLocation=${locationUuid}`;
```

Comment:
For what it's worth, if we end up merging the existing EMR API PR, the endpoint will end up being:

emrapi/inpatient/admissionRequests?admissionLocation=e5db0599-89e8-44fa-bfa2-07e47d63546f

... the following caveats:

* as of now, it won't actually limit via admission location
* it won't yet include transfer requests